content/_ext/yaml_gallery_generator.py

```diff
@@ -143,8 +154,11 @@ def build_from_items(items, filename, display_name, menu_html):
 :card: +my-2
 :img-top-cls: w-75 m-auto p-2
 :body: d-none
+
 {dedent(panels_body)}
 ````
+<div class="backdrop"></div>
+<script src="/_static/custom.js"> </script>
```

Comment:
I'm curious... Is the `html_js_files` in `conf.py` not enough?

```python
html_js_files = ["custom.js"]
```

Reply:
Correct. It does not work if that line 161 isn't there.

Reply:
Interesting... Thank you for the clarification

Reply:
The `html_js_files` in `conf.py` does not seem to be necessary though.